Makefile

```diff
@@ -634,7 +635,8 @@ $(COLLECTED_SOURCES): $(ROOTFS_TAR) $(GOSOURCES)| $(INSTALLER)
 		do								 		   \
 			$(GOSOURCES) sources -d $$(dirname $$i) --recursive --out $(TMP_COLLECTED_SOURCES)/go; \
 		done;	
-	tar cvf - $(TMP_COLLECTED_SOURCES) | gzip -9 - > $(COLLECTED_SOURCES)
+	tar -cf $(COLLECTED_SOURCES) -C $(TMP_COLLECTED_SOURCES) . 
+	bash tools/generate-sources-manifests.sh $(COLLECTED_SOURCES) $(COLLECTED_SOURCES_MANIFESTFILE)
```

Comment:
Can we move this up a line? By which I mean, _before_ creating the tar file with all of the sources, can we generate the manifest, with all of the paths relative to the root (i.e. `$(TMP_COLLECTED_SOURCES)`), and then have the manifest _included_ in the tar file? So right before you tar, you end up with:

```
$(TMP_COLLECTED_SOURCES)/go          # go sources
$(TMP_COLLECTED_SOURCES)/alpine    # alpine sources
$(TMP_COLLECTED_SOURCES)/kernel    # kernel sources
$(TMP_COLLECTED_SOURCES)/manifest.csv
```

and then the tar file includes all of those directories _and_ the `manifest.csv`.

The result of which will be a single tar file, with the manifest inside.

Reply:
I will make the required changes.

Reply:
changes done.